### ISSUE TITLE
Directive vs comments when no space after `::`

### ISSUE THREAD

**Issue by Carreau** [stsewd/tree-sitter-rst#46]
from the https://docutils.sourceforge.io/docs/ref/rst/directives.html 
> followed by the directive type and two colons (collectively, the "directive marker"

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#directives
> followed by the directive type, two colons, **and whitespace** (together called the "directive marker"

Emphasis mine.

**Comment by Carreau** [comment]
Feel free to close, I just wanted to just have a note of it – I think the current behavior is better.

**Comment by stsewd** [comment]
Yeah, I have found several of things like this in the spec that docutils follows loosely. I'm not 100% sure about what to do here... 

- a) Just keep the current behavior and document this under https://github.com/stsewd/tree-sitter-rst#design-notes
- b) See if it's easy to support this and just follow that docutils does instead of what the spec says.

**Comment by Carreau** [comment]
it was suggested to capture the extra space and document and use a query to report the violation.
